packages/bpk-component-checkbox/src/BpkCheckbox.js

```diff
@@ -70,12 +69,10 @@ const BpkCheckbox = props => {
         disabled={disabled}
         aria-label={label}
         aria-invalid={isInvalid}
+        // eslint-disable-next-line no-return-assign, no-param-reassign
+        ref={el => el && (el.indeterminate = indeterminate)}
         {...rest}
       />
-      <Tick
```

Comment:
Removed this as it seemed to have no effect on the field when we are using the svgs this was having no effect.

Verified in all browsers IE 11, Edge, Chrome, Firefox and Safari